english/explore-outside/talk-child/talk-more/nice-talk/help-child/keep-looking/enter-cave/follow-person/follow-person.md

```diff
@@ -0,0 +1 @@
+You use all your strength to swim toward the end of the cave. The person has swam out of site. Can you make it? Do you keep going?
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.